src/components/Books.js

```diff
@@ -0,0 +1,96 @@
+import Form from './Form';
+
+const Books = () => (
+  <div className="booksDiv">
+    <div className="eachBookDiv">
+      <div className="eachBook">
+        <div className="bookDetails">
+          <p className="genre">Action</p>
+          <p className="bookTitle">The Hunger Games</p>
+          <p>Susanne Collins</p>
+          <div className="commentRemoveEditDiv">
+            <p className="commentRemoveEditEffects">Comments</p>
+            <p className="commentRemoveEditEffects">Remove</p>
+            <p className="commentRemoveEditEffects">Edit</p>
+          </div>
+        </div>
+        <div className="completedRing">
+          <div className="ring">.</div>
+          <div>
+            <h2 className="percent">64%</h2>
+            <span>Completed</span>
+          </div>
+        </div>
+        <div className="currentChapter">
+          <div className="line">.</div>
+          <div>
+            <p className="chapterDetails">CURRENT CHAPTER</p>
+            <p className="chapterNumber">Chapter 17</p>
+            <button type="button" className="progressButton">UPDATE PROGRESS</button>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
I noticed the repetition of this particular block of codes in your markup 👀, However, to follow our list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Please let's reduce the number of repeated codes by following the DRY principle (Do not Repeat Yourself) 🤓👍

According to the project requirements, please let's implement the following 🤗

A `<Books>` component should contain the following components:

* `<BookList>`
  * Renders a list of books using the `<Book>` component
* `<Book>`
  * Takes the book data as a `prop` and renders the `title` and the `author`
  * Has a button for removing books 🙂